plugins/calendar-resources/src/components/EventTimeEditor.svelte

```diff
@@ -18,6 +18,7 @@
   import { utcToZonedTime } from 'date-fns-tz'
   import DateEditor from './DateEditor.svelte'
 
+  export let className: string = ''
```

Comment:
Hi @iposho, we are trying to avoid passing CSS class names into components.
Could we please update with a some proper parameter name. Thanks.

Reply:
Hi @haiodo! Sure, how about adding a boolean overflowXAuto parameter? 

`export let overflowXAuto: boolean

...

const applyOverFlowXClass = overflowX ? 'overflow-x-auto' : '';`

And after throwing it into the parent div.

`<div class={`flex-row-center flex-gap-2 ${applyOverFlowXClass}`}>`

Reply:
Hi, yes, should be fine, you could actually use 
```svelte
<div class="flex-row-center flex-gap-2" class:overflow-x-auto={overflowX}>
```

without extra text field.

Reply:
Thank you. Pull request updated.